docs(vault) - update sections for vault superflags (#133)
dgraph-io/dgraph-docs: content/enterprise-features/binary-backups.md

friend: [uid] .
```

> Since you have to do a modification to the schema itself, you need an export.

Reviewer: ```suggestion
Because you have to do a modification to the schema itself, you need an export.
```

Reviewer: Tip: Avoid "Since" in favor of "Because", as doing so makes docs more easily readable for English language learners (and avoids Machine Translation issues, too). Similarly, avoid "Once" in favor of "After".

Restore requests will return immediately without waiting for the operation to
finish.

## Offline restore

> The restore utility is a standalone tool today. A new flag `--encryption_key_file` is added to the restore utility so it can decrypt the backup. This file must contain the same key that was used for encryption during backup. Alternatively, starting with `v20.07.0`, the `vault` superflag can be used to restore a backup.

Reviewer: ```suggestion
The restore utility is now a standalone tool. A new flag, `--encryption_key_file`, is now part of the restore utility, so you can use it to decrypt the backup. The file specified using this flag must contain the same key that was used for encryption during backup. Alternatively, starting with `v20.07.0`, the `vault` superflag can be used to restore a backup.
```

> The `dgraph restore` command restores the postings directory from a previously created backup to a directory in the local filesystem. Restore is intended to restore a backup to a new Dgraph cluster not a currently live one. During a restore, a new Dgraph Zero server may be running to fully restore the backup state.

Reviewer: ```suggestion
You can use the `dgraph restore` command to restore the postings directory from a previously-created backup to a directory in the local filesystem. This command restores a backup to a new Dgraph cluster, so it is not designed to restore a backup to a Dgraph cluster that is currently live. During a restore operation, a new Dgraph Zero server might run to fully restore the backup state.
```

> The `--location` (`-l`) flag specifies a source URI with Dgraph backup objects. This URI supports all the schemes used for backup.

Reviewer: ```suggestion
You can use the `--location` (`-l`) flag to specify a source URI with Dgraph backup objects. This URI supports all the schemes used for backup.
```

> The `--postings` (`-p`) flag sets the directory to which the restored posting directories will be saved. This directory will contain a posting directory for each group in the restored backup.

Reviewer: ```suggestion
You can use the `--postings` (`-p`) flag to set the directory where restored posting directories are saved. This directory contains a posting directory for each group in the restored backup.
```

> The `--zero` (`-z`) flag specifies a Dgraph Zero server address to update the start timestamp and UID lease using the restored version. If no Dgraph Zero server address is passed, the command will complain unless you set the value of the `--force_zero` flag to false. If do not pass a zero value to this command, the timestamp and UID lease must be manually updated through Dgraph Zero server's HTTP 'assign' endpoint using the values printed near the end of the command's output.

Reviewer: ```suggestion
You can use the `--zero` (`-z`) flag to specify a Dgraph Zero server address to update the start timestamp and UID lease using the restored version. If no Dgraph Zero server address is passed, the command will complain unless you set the value of the `--force_zero` flag to false. If do not pass a zero value to this command, you need to manually update the timestamp and UID lease using the Dgraph Zero server's HTTP 'assign' endpoint. The updated values should be those that are printed near the end of the command's output.
```

> The `--backup_id` optional flag specifies the ID of the backup series to restore. A backup series consists of a full backup and all the incremental backups built on top of it. Each time a new full backup is created, a new backup series with a different ID is started. The backup series ID is stored in each `manifest.json` file stored in every backup folder.

Reviewer: ```suggestion
You use the `--backup_id` optional flag to specify the ID of the backup series to restore. A backup series consists of a full backup and all of the incremental backups built on top of it. Each time a new full backup is created, a new backup series with a different ID is started. The backup series ID is stored in each `manifest.json` file stored in each backup folder.
```

> The `--encryption_key_file` flag is required if you took the backup in an encrypted cluster and should point to the location of the same key used to run the cluster.

Reviewer: ```suggestion
You use the `--encryption_key_file` flag in cases where you took the backup in an encrypted cluster. The string for this flag must oint to the location of the same key file used to run the cluster.
```

> The `--vault` [superflag]({{< relref "deploy/cli-command-ref.md" >}}) specifies the [Hashicorp Vault](https://www.vaultproject.io/) server address (`addr`), role id (`role-id-file`), secret id (`secret-id-file`) and field that contains the encryption key (`enc-field`) that was used to encrypt the backup.

Reviewer: ```suggestion
You use the `--vault` [superflag]({{< relref "deploy/cli-command-ref.md" >}}) to specify the [Hashicorp Vault](https://www.vaultproject.io/) server address (`addr`), role id (`role-id-file`), secret id (`secret-id-file`) and the field that contains the encryption key (`enc-field`) that was used to encrypt the backup.
```

> The restore feature will create a cluster with as many groups as the original cluster had at the time of the last backup. For each group, `dgraph restore` creates a posting directory `p<N>` corresponding to the backup group ID. For example, a backup for Dgraph Alpha group 2 would have the name `.../r32-g2.backup` and would be loaded to posting directory `p2`.

Reviewer: ```suggestion
The restore feature creates a cluster with as many groups as the original cluster had at the time of the last backup. For each group, `dgraph restore` creates a posting directory (`p<N>`) that corresponds to the backup group ID. For example, a backup for Dgraph Alpha group 2 would have the name `.../r32-g2.backup` and would be loaded to posting directory `p2`.
```
